src/codegen-stubs.c

```diff
@@ -138,3 +138,28 @@ JL_DLLEXPORT void LLVMExtraAddGCInvariantVerifierPass_fallback(void *PM, bool_t
 JL_DLLEXPORT void LLVMExtraAddDemoteFloat16Pass_fallback(void *PM) UNAVAILABLE
 
 JL_DLLEXPORT void LLVMExtraAddCPUFeaturesPass_fallback(void *PM) UNAVAILABLE
+
+//LLVM C api to the julia JIT
+JL_DLLEXPORT void* JLJITGetLLVMOrcExecutionSession_fallback(void* JIT) UNAVAILABLE
```

Comment:
Should we call these LLVMExtra as the other C-API pieces above?

Reply:
Maybe? The reason I used a different one is that they all refer specifically to the julia JIT. But we could do something like `LLVMExtraJLJIT...`

Reply:
I'd treat this differently from LLVMExtra, which are extensions to the LLVM C APIs. This is very specifically related to interacting with Julia's compiler.

Reply:
Do you have a suggestion for a name? I was using JLJIT to pay homage to LLJIT but if you have any ideas i'm all ears.